packages/language-server/src/completion/completions.ts

```diff
@@ -591,6 +591,16 @@ function getDefaultValues({
       break
   }
 
+  const isScalarList = fieldType.endsWith('[]')
+  if (isScalarList) {
+    suggestions.unshift({
+      label: '[]',
+      insertText: '[$0]',
+      documentation: 'Set a default value on the list field',
+      kind: CompletionItemKind.Value,
+    })
```

Comment:
```suggestion
    suggestions.unshift({
      label: '[]',
      insertText: '[$0]',
      insertTextFormat: InsertTextFormat.Snippet,
      documentation: 'Set a default value on the list field',
      kind: CompletionItemKind.Value,
    })
```

fix, currently autocompletes with `[$0]` instead of moving the cursor to `$0`